data-esnext.js

```diff
@@ -481,25 +481,32 @@ exports.tests = [
   }
 },
 {
-  name: 'weak references',
+  name: 'WeakReferences',
   spec: 'https://github.com/tc39/proposal-weakrefs',
-  category: STAGE2,
+  category: STAGE3,
   significance: 'large',
-  exec: function(){/*
-    var O = {};
-    var weakref = System.makeWeakRef(O);
-    var works = weakref.get() === O;
-    weakref.clear();
-    return works && weakref.get() === undefined;
-  */},
-  res : {
-    ie11: false,
-    firefox2: false,
-    opera10_50: false,
-    chrome65: false,
-    duktape2_0: false,
-    graalvm: false,
-  }
+  subtests: [
+    {
+      name: 'Weak references',
+      spec: 'https://github.com/tc39/proposal-weakrefs#weak-references',
+      exec: function(){/*
+        var O = {};
+        var weakref = new WeakRef(O);
+        var works = weakref.deref() === O;
+        weakref.clear();
+        return works && weakref.deref() === undefined;
+      */},
+      res : {
+        ie11: false,
+        firefox2: false,
+        opera10_50: false,
+        chrome65: false,
+        chrome77: {val: 'flagged', note_id: "chrome-weakrefs", note_html: "Available behind the <a href='https://bugs.chromium.org/p/v8/issues/detail?id=8179'><code>--harmony-weak-refs</code></a> flag in V8."},
```

Comment:
It's implemented in Chrome 74 with flag "--js-flags=--harmony-weak-refs --expose-gc"

Reply:
I'll fix it.

Reply:
Done.